garage/replay_buffer/replay_buffer.py

```diff
@@ -0,0 +1,72 @@
+"""
```

Comment:
Wasn't the replay buffer already added?

Reply:
yes, must've been added in when I rebased?